sim/side.ts

```diff
@@ -629,19 +629,25 @@ export class Side {
 
 			switch (choiceType) {
 			case 'move':
-				let targetLoc = 0;
+				const original = data;
+				const error = () => this.emitChoiceError(`Conflicting arguments for "move": ${original}`);
+				let targetLoc: number | undefined;
 				let megaOrZ = '';
 				while (true) {
 					if (/\s-?[1-3]$/.test(data)) {
+						if (targetLoc !== undefined) error();
```

Comment:
This is going to be a mess when in conjunction with `Conversion 2`

Reply:
How did we actually do this in the past, anyway?

Reply:
It looks like neither Conversion nor Conversion 2 are targeted, but various consumers might want to send a target anyway.

Reply:
If a Pokémon in move slot two sends `move Conversion 2`, and has both `Conversion` and `Conversion 2` in its moveset, what should happen?

Reply:
IIRC, its position would be `-2`, not `2`. `2` would be targetting the opponent, which would be an invalid target anyway (regardless of whether its `Conversion` or `Conversion 2`).

I think in the `Conversion[ 2]` case we should just special case it to ignore targetting information, because theyre both `target: self`

Reply:
Oh, good point. That sounds good to me.